test/teiserver_web/tachyon/system_test.exs

```diff
@@ -0,0 +1,40 @@
+defmodule TeiserverWeb.Tachyon.SystemTest do
+  use TeiserverWeb.ConnCase
+  alias Teiserver.Support.Tachyon
+  alias Teiserver.OAuthFixtures
+
+  describe "server stats" do
+    defp setup_app(_context) do
+      owner = Central.Helpers.GeneralTestLib.make_user(%{"data" => %{"roles" => ["Verified"]}})
+
+      app =
+        OAuthFixtures.app_attrs(owner.id)
+        |> Map.put(:uid, UUID.uuid4())
+        |> OAuthFixtures.create_app()
+
+      {:ok, app: app}
+    end
+
+    defp setup_user(app) do
+      user = Central.Helpers.GeneralTestLib.make_user(%{"data" => %{"roles" => ["Verified"]}})
+      token = OAuthFixtures.token_attrs(user.id, app) |> OAuthFixtures.create_token()
+      client = Tachyon.connect(token)
+      {:ok, %{user: user, token: token, client: client}}
+    end
+
+    setup [:setup_app]
+
+    test "works", %{app: app} do
+      {:ok, %{user: _user1, client: client1}} = setup_user(app)
```

Comment:
nitpick: you don't need to pattern match on `user` if you don't use it.

Reply:
Ah, copy pasted this one from pairing test without checking... fixed.